scripts/scenarios/donuts/tree-test.ts

```diff
@@ -18,19 +23,49 @@ export const tree: CardTree = {
         card: quickCard("A:L", "A:L:L", "A:L:R"),
         left: {
             card: quickCard("A:L:L", "A:L:L:L", "A:L:L:R"),
+            modifiers: addModifier({ left: 1 }),
+            left: {
+                modifiers: addModifier({ left: 1 }),
```

Comment:
Good idea to add a counter to verify that the different states has been reached. And even for the leaf nodes.